providers/apis/binance/utils.go

```diff
@@ -33,7 +33,7 @@ var DefaultNonUSAPIConfig = config.APIConfig{
 	Atomic:           true,
 	Enabled:          true,
 	Timeout:          3000 * time.Millisecond,
-	Interval:         750 * time.Millisecond,
+	Interval:         1000 * time.Millisecond,
```

Comment:
Why are we messing with provider intervals in this PR?

Reply:
accident, was just about to change it back

Reply:
updated